.github/workflows/bi-weekly-automated-model-build.yml

```diff
@@ -0,0 +1,26 @@
+name: nightly-clear-model-instances
+
+on:
```

Comment:
Let's add a dispatch trigger here just for convenience

Reply:
Already there